content/news/2024/01/31/pybamm-24.1.md

```diff
@@ -0,0 +1,47 @@
+---
+title: PyBaMM 24.1 has been released!
+summary: PyBaMM version 24.1 has now been released with several new features and improvements.
+date: 2024-01-31
+toc: true
+shortcutDepth: 1
+---
+
+PyBaMM 24.1 has now been released! This release continues the previous release schedule conforming to three releases a year and marks the first release of 2024. We would like to thank all the contributors who made this release and the previous three pre-releases possible.
```

Comment:
Maybe link all the contributors (or the teams page) here?

Reply:
Added a link to the teams page in https://github.com/pybamm-team/pybamm.org/commit/4662b07da2ddb2d8f77154ce63e6bf809acd5233